.travis.yml

```diff
@@ -157,3 +157,14 @@ matrix:
         - mbedtools deploy
         - mbedtools compile -t GCC_ARM -m K64F
         - ccache -s
+
+    - <<: *mbed-tools-test
+      name: "Test deploy command"
```

Comment:
Could you explain what this test is doing for the "name"?

It looks like you are not testing deploy how the user would use it (as you still clone mbed-os manually with git).

Reply:
Idea is to clone some version of Mbed OS to unknown commit and ask `mbed deploy` to update the libraries

Reply:
Explain it in the name field please, not on the PR.

Reply:
I've described it.